pkg/list/list_vendor_test.go

```
@@ -0,0 +1,263 @@
+	}
+	vendorManifestBytes, err := yaml.Marshal(vendorManifest)
+	require.NoError(t, err)
+	err = os.WriteFile(vendorManifestFile, vendorManifestBytes, 0o644)
+	require.NoError(t, err)
+
+	// Change to the temporary directory for testing
+	currentDir, err := os.Getwd()
+	require.NoError(t, err)
+	err = os.Chdir(tmpDir)
+	require.NoError(t, err)
+	defer os.Chdir(currentDir)
```

golangci-lint (1.62.2)

Error return value of `os.Chdir` is not checked

(errcheck)